apps/trading/pages/styles.css

```diff
@@ -22,3 +22,17 @@ html.dark {
 .border-default {
   @apply border-neutral-300 dark:border-neutral-600;
 }
+
+.container [data-theme='dark'] {
```

Comment:
this does not yet override the pennant charts. I need to figure out how to do it. 
https://pennant.vega.xyz/docs/colors/#using-custom-colors